app.js

```diff
@@ -0,0 +1,63 @@
+
+var myNum = {
```

Comment:
This is a very clever solution! However, the idea is that an object represents one real-world object. In this lab, one object should represent one store. This means that each object should have stored its own min, max, ave, and city instead of having one object that contains everything.